app/models/project.rb

```diff
@@ -39,6 +55,29 @@ def samples
     nil
   end
 
+  def cleaned_project_name
+    "project-#{name.downcase.split(' ').join('_')}"
+  end
+
+  def host_gene_counts_from_params(params)
```

Comment:
maybe more error handling in this function?

Reply:
ok. I'm making it admin only for now. We can add more error handling once we start showing it to users